example-files/notebooks/mango-python-alignment.ipynb

```diff
@@ -1,11 +1,19 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Is the modification of this file required for docs? If not, can we revert it to the master?